configs/experiment/im2im/segmentation_plugin.yaml

```diff
@@ -0,0 +1,52 @@
+# @package _global_
+# to execute this experiment run:
+# python train.py experiment=example
+defaults:
+  - override /data: im2im/segmentation_plugin.yaml
```

Comment:
These are new to support the API. I feel like we talked about storing configs in cytodl instead of the plugin repo, but happy to change if I'm misremembering. 

I've been calling this section im2im because it refers to the image-to-image portion of cytodl

Reply:
> I feel like we talked about storing configs in cytodl instead of the plugin repo
Yes for sure.  I think ideally, the plugin just calls functions, and doesnt need to know anything about config files (of even the names of keys).  Reason: this would make integration testing more effective.